runtime/oti/j9nonbuilder.h

```diff
@@ -2954,6 +2954,11 @@ typedef struct J9Class {
 #define J9INTERFACECLASS_ITABLEMETHODCOUNT(clazz) ((clazz)->totalInstanceSize)
 #define J9INTERFACECLASS_SET_ITABLEMETHODCOUNT(clazz, value) (clazz)->totalInstanceSize = (value)
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+#define J9ARRAYCLASS_SET_STRIDE(clazz, strideLength) *((UDATA*)&(clazz->flattenedClassCache)) = (UDATA) strideLength;
```

Comment:
semi-colons are not needed for both macros

Reply:
I would prefer as cast to the final type, rather than casting the result location. Casting pointers to odd things can result in aliasing errors.